components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -118,6 +118,8 @@ struct SearchStatusEvent {
 
 struct CompletionEvent {
   string completion;
+  // Whether the completion was generated from Brave Search SERP.
+  bool from_brave_search_SERP = false;
 };
```

Comment:
Should this extra field be on ConversationTurn? Can a turn's event be consumed from something external, but not the whole entry? Seems simpler to have it on ConversationTurn and leave future possibility for consuming different kind of events (e.g. sources)

Reply:
Addressed in https://github.com/brave/brave-core/pull/25257/commits/138ebaf68057d3926f90b4bfeafa8cab2fd660b0.